### PR TITLE
Update TE-3.5 README

### PR DESCRIPTION
**Pull request by xw-g**
* In the case of FIB ACK, RIB ACK is optional. gRIBI does allow a device to only return FIB ACK (skip RIB_ACK) after a success FIB programming. This is also aligned with current implementation that checks either FIB ACK or RIB ACK (with flag), but not both.  
* Remove case#2:
	* If case#2 means IPv4Entry -> NHG -> NH, then it's already covered by case#4.
	* If case#2 means hierarchical resolution, then it's covered by TE-3.1
	* case#2 is not implemented anyway.

### CODE REVIEW

**Review by liulk — APPROVED** [review]

File: feature/gribi/ate_tests/ordering_ack_test/README.md [hunk 1]
```diff
@@ -11,11 +11,8 @@ Ensure that acknowledgements are sent as is expected by gRIBI controller.
     the requested `ack_type`.
 *   Install the following entries and determine whether the expected result is
     observed:
-    *   A `NextHopGroup` referencing a `NextHop` is responded to with RIB+FIB
+    *   A `NextHopGroup` referencing a `NextHop` is responded to with FIB
         ACK, and is reported through the AFT telemetry.
-    *   A `NextHopGroup` referencing a `NextHop` with a referencing `IPv4Entry`
```
Comment (liulk) [inline feature/gribi/ate_tests/ordering_ack_test/README.md:16]: Wait why is this deleted?

**Review by liulk — CHANGES_REQUESTED** [review]

File: feature/gribi/ate_tests/ordering_ack_test/README.md (hunk 1 above)
Comment (xw-g) [reply feature/gribi/ate_tests/ordering_ack_test/README.md:16]: See the PR description?

**Review by liulk — APPROVED** [review]
